Veteran and Elite Sight Ranges (#1656)
Vinifera-Developers/Vinifera: docs/Miscellaneous.md

- Hospitals and armories can now accept multiple infantry, which will form a queue around them. Units will go in one at a time.
- If charges (ammo) deplete while units are still waiting in the queue, remaining units will be dismissed and be ordered to go to the respective rally point instead.

## Veteran and Elite Sights

> - Technos can now be assigned specific values to use as their sight range when they are Veterans and Elites, respectively.

Reviewer: ```suggestion
- Technos can now be given specific values to use as their sight range when they are Veteran or Elite.
```

- When a key is not specified, technos will fall back to the lower level sight. For example, if `EliteSight` is not specified but `VeteranSight` is, a techno that is Elite would use `VeteranSight`. If it's also not specified, it will fall back to `Sight`.

> In `RULES.INI` or a scenario file:

Reviewer: ```suggestion
In `RULES.INI`:
```

This is true for any rules key, no need to mention it.
